release-canary.js

```diff
@@ -0,0 +1,46 @@
+const semver = require('semver');
```

Comment:
Having a `.js` script to do this is not out of the ordinary, that's what they do in the React package to release canary:

* https://github.com/facebook/react/blob/main/scripts/release/build-release-locally-commands/update-version-numbers.js
* https://github.com/facebook/react/blob/ec15267a001086deb4ab5412d3f8b7e13573d6a5/scripts/release/utils.js#L223